src/multivariate/mvlogitnormal.jl

```diff
@@ -0,0 +1,150 @@
+"""
+    MvLogitNormal{<:AbstractMvNormal}
+
+The [multivariate logit-normal distribution](https://en.wikipedia.org/wiki/Logit-normal_distribution#Multivariate_generalization)
+is the multivariate generalization of [`LogitNormal`](@ref).
+
+It is a more general alternative to [`Dirichlet`](@ref) for probability vectors.
+
+If ``\\mathbf{y} \\sim \\mathrm{MvNormal}(\\boldsymbol{\\mu}, \\boldsymbol{\\Sigma})`` is a
+length ``d-1`` vector, then
+```math
+\\mathbf{x} = \\operatorname{softmax}\\left(\\begin{bmatrix}\\mathbf{y} \\\\ 0 \\end{bmatrix}\\right) \\sim \\mathrm{MvLogitNormal}(\\boldsymbol{\\mu}, \\boldsymbol{\\Sigma})
+```
+is a length ``d`` probability vector.
+
+```julia
+MvLogitNormal(μ, Σ)                 # MvLogitNormal with y ~ MvNormal(μ, Σ)
+MvLogitNormal(MvNormal(μ, Σ))       # same as above
+MvLogitNormal{MvNormal}(μ, Σ)       # same as above
+MvLogitNormal(MvNormalCanon(μ, J))  # MvLogitNormal with y ~ MvNormalCanon(μ, J)
+MvLogitNormal{MvNormalCanon}(μ, J)  # same as above
+```
+
+# Fields
+
+- `normal::AbstractMvNormal`: contains the ``d-1``-dimensional distribution of ``y``
+"""
+struct MvLogitNormal{D<:AbstractMvNormal} <: ContinuousMultivariateDistribution
+    normal::D
+    MvLogitNormal{D}(normal::D) where {D<:AbstractMvNormal} = new{D}(normal)
+end
+function MvLogitNormal{D}(args...) where {D<:AbstractMvNormal}
+    normal = D(args...)
+    return MvLogitNormal{typeof(normal)}(normal)
+end
```

Comment:
I'm not sure if this method is a good idea. Maybe at least initially it could be omitted?
```suggestion
```
My main concern is that it allows users to input all kinds of subtypes of `AbstractMvNormal`, possibly also with type parameters, and it is not clear in which cases these will work and when the desired type is actually respected (similar to the confusions/inconsistencies of the type argument in the `fit` methods). To me the alternative `MvLogitNormal(::AbstractMvNormal)` seems to be not susceptible to such issues, and the type version is not even shorter.